src/openloco/objects/industry_object.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "../localisation/stringmgr.h"
+
+namespace openloco
+{
+    struct industry_object
+    {
+        string_id name;
+        uint8_t pad_02[0x11 - 0x02];
+        uint8_t var_11;
+        uint8_t pad_12[0xEA - 0x12];
+        uint32_t var_E4;
```

Comment:
Locotools has this marked as flags with the following flags  (1<<17) "needall", "canincreaseproduction", "candecreaseproduction",